pylint/config/find_default_config_files.py

```diff
@@ -24,40 +24,45 @@ def _cfg_has_config(path):
     return any(section.startswith("pylint.") for section in parser.sections())
 
 
-def find_default_config_files():
-    """Find all possible config files."""
-    rc_names = ("pylintrc", ".pylintrc")
-    config_names = rc_names + ("pyproject.toml", "setup.cfg")
+def _get_config_paths(curdir):
```

Comment:
Could you add typing on the modified functions, please ? :) 
```suggestion
def _get_config_paths(curdir: Union[Path, str]) -> List[Path]:
```